backend/services/theme_service.py

```
@@ -717,12 +717,12 @@ async def _generate_with_multi_api_controlnet(
             
             face_preserved_bytes = await self.runware_service.generate_with_face_reference(
                 face_image_bytes=base_image_bytes,
-                prompt=face_preservation_prompt,
-                negative_prompt="different face, changed face, face swap, wrong identity",
+                prompt=final_prompt,  # Use full theme prompt instead of minimal
```

Ruff (0.12.2)

Undefined name `final_prompt`

(F821)